ci-operator/step-registry/hypershift/azure/setup-nested-management-cluster/hypershift-azure-setup-nested-management-cluster-chain.yaml

```diff
@@ -0,0 +1,88 @@
+chain:
+  as: hypershift-azure-setup-nested-management-cluster
+  steps:
+    - as: create-management-cluster
+      cli: latest
+      env:
+        - name: HYPERSHIFT_NODE_COUNT
+          default: "3"
+          documentation: "The number nodes for the cluster."
+        - name: HYPERSHIFT_INSTANCE_TYPE
+          default: "Standard_D4s_v4"
+          documentation: "Instance type for the cluster nodes"
+        - name: HYPERSHIFT_NAMESPACE
+          default: "clusters"
+          documentation: "The Namespace where to create the HostedCluster and NodePools"
+        - name: HYPERSHIFT_BASE_DOMAIN
+          default: ""
+          documentation: "The cluster's FQDN will be a subdomain of the base domain."
+      commands: |-
+        DEFAULT_BASE_DOMAIN="hypershift.azure.devcluster.openshift.com"
+        
+        DOMAIN=${HYPERSHIFT_BASE_DOMAIN:-$DEFAULT_BASE_DOMAIN}
+        
+        DEFAULT_RELEASE_STREAM=4.16.0-0.nightly
```

Comment:
why nightly?

Reply:
this is for the management cluster, right? why would we want to catch errors related to the mgmt cluster in e2e?

Reply:
My thought while setting this up was catch out issues with creating hc but thinking of it now the e2e covers that. I will change it to use 4-dev-preview

Reply:
Although we do use 4.16 nightlies in our AWS e2e aswell so don't see any harm in leaving it as is.

Reply:
I think we should do whatever we do for AWS e2e to be consistent

Reply:
I think there was an  issue in AWS that made us switch to nightly temporary. I think we should be using `4-stable`.
@sjenning wdyt?

Reply:
In general, I would rather not copy/paste this rather large chain.  How does it differ from the AWS version?  Could we do the `CLOUD_PROVIDER` if/else thing you do on the hypershift-install chain?